docs/src/main/asciidoc/opentelemetry.adoc

```diff
@@ -628,7 +628,7 @@ The exporter is automatically wired with CDI, that's why the `quarkus.otel.trace
 The `quarkus.otel.exporter.otlp.traces.protocol` default to `grpc` and `http/protobuf` can also be used.
 
 === On Quarkiverse
-Additional exporters will be available in the Quarkiverse https://github.com/quarkiverse/quarkus-opentelemetry-exporter/blob/main/README.md[quarkus-opentelemetry-exporter] project.
+Additional exporters will be available in the Quarkiverse https://docs.quarkiverse.io/quarkus-opentelemetry-exporter/dev/quarkus-opentelemetry-exporter-azure.html[quarkus-opentelemetry-exporter] project.
```

Comment:
Shouldn't it be https://docs.quarkiverse.io/quarkus-opentelemetry-exporter/dev/index.html instead?

Reply:
Yes! Sorry, a bad copy paste